src/models/index.ts

```diff
@@ -1,9 +1,6 @@
-import { Account } from './account';
-import { BaseModel } from './base-model';
-import { ModelFactory } from './model-factory';
+import { Account } from './account'
+import { BaseModel } from './base-model'
+import { ModelFactory } from './model-factory'
+import { TodoItem } from './todo-item'
 
-export {
-  Account,
-  BaseModel,
-  ModelFactory,
-};
+export { Account, BaseModel, ModelFactory, TodoItem }
```

Comment:
Style: Exports should be consistent, please define your export items in new line.

```js
export {
  Account,
  BaseModel,
  ...
}
```